src/libraries/Microsoft.Extensions.Configuration.Json/src/Microsoft.Extensions.Configuration.Json.csproj

```diff
@@ -16,7 +16,7 @@
     <Reference Include="System.Text.Json" />
   </ItemGroup>
 
-  <ItemGroup Condition="$(TargetFramework.StartsWith('netcoreapp'))">
+  <ItemGroup Condition="$(TargetFramework.StartsWith('$(NetCoreAppCurrent)'))">
```

Comment:
Should this instead test for equality?

Reply:
No, because the TargetFramework is overloaded with the OS: `netcoreapp5.0-Windows_NT`.
I will create a doc after merging this which describes when to use what. Basically the recommendation is:
1. Use an equality check if the TargetFramework isn't overloaded with the OS portion.
2. Use a StartsWith when you want to test for .NETStandard or .NETFramework
3. Use a StartsWith if the TargetFramework is overloaded with the OS portion.
4. Use negations if that makes the conditions easier.